modules/ROOT/pages/_partials/data-sync/js-client-offline.adoc

```diff
@@ -6,20 +6,16 @@
 
 Your mobile app can run offline and allow users to query and create mutations using the @aerogear/voyager-client module. 
 
-As shown in the diagram below, all queries are performed against the cache, a mutation store supports offline mutations.
-
-image::datasync-features.png[]
-
-NOTE: The mutation store is sometimes referred to as the offline store.
+All queries are performed against the cache, a mutation store (or offline store) supports offline mutations.
 
 If a client goes offline for a long period of time, the mutation store negotiates local updates with the server using conflict resolution strategies.
 
-When a client comes online again, the mutations are replicated back to the server, as shown in the diagram below:
-
-image::datasync-going_offline.png[]
+When a client comes online again, the mutations are replicated back to the server.
 
 Developers can attach listeners to get notifications about updates applied on the server or failing, and take appropriate actions.
 
+NOTE: If your business logic requires that certain mutations should only occur when online, you can include that logic using the `@onlineOnly` directive
```

Comment:
Shouldn't there be an example of using the @onlineOnly directive here?  There is an example in the current version ( https://mobile-docs.netlify.com/aerogear/latest/ds-offline/#ensuring-specified-mutations-are-performed-online-only )

Reply:
remove note